multicast/__main__.py

```
@@ -134,9 +134,9 @@
 
 try:
 	from . import sys
-except ImportError as impErr:
+except ImportError as baton:
 	# Throw more relevant Error
-	raise ImportError(str("[CWE-440] Error Importing Python")) from impErr
+	raise ImportError(str("[CWE-440] Error Importing Python")) from baton
```

Ruff (0.8.2)

Unnecessary `str` call (rewrite as a literal)

Replace with string literal

(UP018)